cmake/curl.cmake

```diff
@@ -51,3 +51,15 @@ MACRO(MYSQL_CHECK_CURL)
    ENDIF()
   ENDIF()
 ENDMACRO()
+
```

Comment:
@robgolebiowski Once again, do we really need this RTMP checking? At least on Ubuntu Bionic unit tests work fine even though curl library there depends on rtmp.

Reply:
@percona-ysorokin It is possible that they fixed it already. I remember it was failing a lot on centos6. There is JIRA for it PS-4757. Will try to take care of that as a background task.